src/lib/state.test.js

```diff
@@ -0,0 +1,57 @@
+import state from './state';
+
+describe('state', () => {
```

Comment:
No need for this `describe`, since the filename's already saying we're testing `state`